src/Components/Footer/Footer.js

```diff
@@ -1,7 +1,125 @@
-import React from 'react';
+/*eslint-disable */
+import { useState } from 'react';
+import { withRouter, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import './Footer.scss';
+import { AiFillYoutube } from 'react-icons/ai';
+import { AiFillInstagram } from 'react-icons/ai';
+import { FaFacebookF } from 'react-icons/fa';
+import { AiOutlineComment } from 'react-icons/ai';
```

Comment:
react-icons도 라이브러리이기 때문에 import 순서를 아래와 같이 변경해주시면 더 컨벤션에 적절합니다!
```suggestion
import { useState } from 'react';
import { withRouter, Link } from 'react-router-dom';
import styled from 'styled-components';
import { AiFillYoutube } from 'react-icons/ai';
import { AiFillInstagram } from 'react-icons/ai';
import { FaFacebookF } from 'react-icons/fa';
import { AiOutlineComment } from 'react-icons/ai';
import './Footer.scss';
```

Reply:
참조 및 수정하였습니다 +_+!